README.md

```diff
@@ -65,6 +65,8 @@ python sounding_to_netcdf.py
 deactivate
 
 ```
+, with a Python version >= 3.7 and < 3.11.
```

Comment:
What restricts us to 3.11?

Reply:
I forget which package it was, but there was one dependency that explicitly stated this range.